src/sprout/sprout_load.php

```diff
@@ -43,6 +43,8 @@
 
 Register::sitemapGen('\\Sprout\\Helpers\\SitemapGenPages');
 
+Register::searchHandler('Sprout\\Controllers\\PageController', 'page_keywords', ['main.active = 1', 'main.show_in_nav = 1']);
```

Comment:
or HERE you could call `Register::searchHandler` for every item in old config instead.